tests/lib/pkgdb.sh

```diff
@@ -619,6 +619,7 @@ pkg_dependencies_ubuntu_classic(){
                 qemu-kvm
                 qemu-utils
                 shellcheck
+                xxd
```

Comment:
I think we are replacing with od that is more readily available and we have already merged a PR replacing this. But maybe it is still used in other tests. If that is the case, maybe add a TODO about replacing in the future.

Reply:
fixed

Reply:
@sergiocazzolato I don't think this is fixed yet, perhaps rebase on master and drop this?